core-bundle/contao/templates/backend/be_main.html5

```diff
@@ -33,8 +33,6 @@
       $objCombiner->add('assets/chosen/js/chosen.min.js');
       $objCombiner->add('assets/simplemodal/js/simplemodal.min.js');
```

Comment:
What about the other ones? Can't we add them via `asset()` too?

Reply:
Yes, but then we would end up with a lot of single requests instead of just one for the combined file.

Reply:
So? It's even better with single requests. Why would I download a file that contains the scripts for chosen, simplemodal, datepicker etc. combined? Chances that only one of them changes during a Contao update are pretty high. The way it is now it would generate a new combined file forcing me to download it all again.
Yes, I agree it's still better to combine the stuff that belongs together in one file, even with http2/3. But these files are completely independent from each other. Chosen is not related to simplemodal whatsoever. Imho, we should not combine here.

Reply:
Agreed. But let‘s change this in a separate pull request, because it is not really related to this one.

Reply:
Okay 👍

Reply:
See #5626.